src/pages/Dashboard.vue

```diff
@@ -97,6 +97,7 @@ export default defineComponent({
       this.global.status.message = lang.syncedMsg
       this.checkNodeAndNetwork()
       this.checkFarmerAndPlot()
+      this.client.disconnectPublicApi()
```

Comment:
Why does this component need to connect to public API at all?

Reply:
See the below comment. 
`checkNodeandNetwork` checks for last block from the publicRPC to inform the user on sync status.
`checkFarmerAndPlot` retrieves the blockchain size from the publicRPC, but it will be gone after partial replica PR (we will be querying the plot size from config, because user gets to choose it)

Reply:
Does this mean public RPC connection can be established and closed in 2 places, but it is the same global variable? Looks like a recipe for confusing bugs.